service/matching/version_rule_test.go

```diff
@@ -220,33 +218,33 @@ func TestInsertAssignmentRuleBasic(t *testing.T) {
 	rule1 := mkAssignmentRule("1", nil)
 	data, err := insertAssignmentRule(rule1, initialData, clock, 0, maxRules)
 	assert.NoError(t, err)
-	expected.AssignmentRules = slices.Insert(expected.AssignmentRules, 0, mkAssignmentRulePersistence(rule1, clock, nil))
+	expected.AssignmentRules = slices.Insert(expected.AssignmentRules, 0, mkAssignmentRulePersistence(mkAssignmentRule("1", mkNewAssignmentPercentageRamp(100)), clock, nil))
```

Comment:
Nit: maybe we can pass only the number to `mkAssignmentRule` and let it call `mkNewAssignmentPercentageRamp`.

Reply:
nvm, you want to be able to pass nil.

Reply:
yeah i like that. sure why not